packages/catalog-process/src/utilities/versionGenerator.ts

```diff
@@ -14,7 +14,7 @@ export const nextDescriptorVersion = (eservice: EService): string => {
     }
 
     return currentVersionNumber.data > max ? currentVersionNumber.data : max;
-  }, 0);
+  }, 1);
```

Comment:
Bug fix. This function is used to evaluate the descriptor version during the `createDescriptor` operation. Those versions should start from 1. Instead, here the versions started from 0.